acceptance/bundle/python/restricted-execution/mutators.py

```diff
@@ -0,0 +1,14 @@
+from dataclasses import replace
+from databricks.bundles.jobs import Job
+from databricks.bundles.core import job_mutator, Bundle
+import os
+
+
+@job_mutator
+def read_envs(bundle: Bundle, job: Job) -> Job:
+    # Reading from envs to ensure that the envs are accessible for this mutator
+    # (which this test case is verifying that they are not)
```

Comment:
```
(which this test case is verifying that they are not)
```

This part is a little hard to read. Change this to "this test ensures environment variables are not readable by pydabs in restriction execution mode because pydabs mutators are not executed in the first place?"